CFraction/CFraction/CFraction.cpp

```diff
@@ -0,0 +1,193 @@
+//		{
+//			continue;
+//		}
+//		else if()
+//	}
+//}
+
+void Fraction::set_numerator(int numerator)
+{
+	_numerator = numerator;
+}
+void Fraction::set_denominator(int denumirator)
+{
+	_denominator = denumirator;
+}
+int Fraction::get_numerator()
+{
+	return _numerator;
+}
+int Fraction::get_denuminator()
+{
+	return _denominator;
+}
+
+void Fraction::print()
+{
+	std::cout << _numerator << '/' << _denominator << '\n';
+}
+void Fraction::Do_easier()
+{
+	int m = _denominator;
+	int n = _numerator;
+	int r = m % n;
+	while (r != 0)
+	{
+		m = n;
+		n = r;
+		r = m % n;
+	}
+	_denominator /= n;
+	_numerator /= n;
+}
+
+Fraction Fraction::operator=(const Fraction& fr)
+{
+	if (this != &fr)
+	{
+		_denominator = fr._denominator;
+		_numerator = fr._numerator;
+	}
+	else { return *this; }
+}
+
+bool Fraction::operator!=(const Fraction& fr)
+{
+	return (_denominator != fr._denominator && _numerator != fr._numerator);
+}
+
+bool Fraction::operator>=(const Fraction& fr)
+{
+	return ((_denominator == fr._denominator && _numerator >= fr._numerator) || (_denominator <= fr._denominator));
+}
+bool Fraction::operator<=(const Fraction& fr)
+{
+	return((_denominator == fr._denominator && _numerator <= fr._numerator) || (_denominator >= fr._denominator));
+}
+bool Fraction::operator<(const Fraction& fr)
+{
+	return((_denominator == fr._denominator && _numerator < fr._numerator) || (_denominator > fr._denominator));
+}
+bool Fraction::operator>(const Fraction& fr)
+{
+	return((_denominator == fr._denominator && _numerator > fr._numerator) || (_denominator < fr._denominator));
+}
+
+Fraction Fraction::operator+(const Fraction& fr1)
+{
+	Fraction Res(*this);
+	if (Res._denominator == fr1._denominator)
+	{
+		Res._numerator += fr1._numerator;
+	}
+	else
+	{
+		Res._numerator = (Res._numerator * fr1._denominator) + (fr1._numerator * Res._denominator);
+		Res._denominator = Res._denominator * fr1._denominator;
+	}
+	Res.Do_easier();
+	return Res;
+}
+
+void Fraction::operator+=(const Fraction& fr1)
+{
+	if (_denominator == fr1._denominator)
+	{
+		_numerator += fr1._numerator;
+	}
+	else
+	{
+		_numerator = (_numerator * fr1._denominator) + (fr1._numerator * _denominator);
+		_denominator = _denominator * fr1._denominator;
+	}
+	this->Do_easier();
+}
+
+Fraction Fraction::operator-(const Fraction& fr)
+{
+	Fraction Res(*this);
+	if (Res._denominator == fr._denominator)
+	{
+		Res._numerator -= fr._numerator;
+	}
+	else
+	{
+		Res._numerator = (Res._numerator * fr._denominator) - (fr._numerator * Res._denominator);
+		Res._denominator = Res._denominator * fr._denominator;
+	}
+	Res.Do_easier();
+	return Res;
+}
+
+void Fraction::operator-=(const Fraction& fr1)
+{
+	if (_denominator == fr1._denominator)
+	{
+		_numerator -= fr1._numerator;
+	}
+	else
+	{
+		_numerator = (_numerator * fr1._denominator) - (fr1._numerator * _denominator);
+		_denominator = _denominator * fr1._denominator;
+	}
+	this->Do_easier();
+}
+
+Fraction Fraction::operator*(const Fraction& fr)
```

Comment:
`2/1`*`1/2`=`2/2` — do simplification here too

Reply:
I already wrote siplification in this function)